src/App.js

```diff
@@ -76,4 +94,15 @@ export default function App() {
   function removeFromFavorites(id) {
     setFavoriteCrops(favoriteCrops.filter(favCrop => favCrop.id !== id))
   }
+
+  function addTask({ date, tasks, nameOfCrop, id }) {
+    setTaskList([{ date, tasks, nameOfCrop, id }, ...taskList])
+  }
+
+  function deleteTask(id) {
+    const isInTaskList = taskList.some(task => task.id === id)
+    if (isInTaskList) {
+      setTaskList(taskList.filter(task => task.id !== id))
+    }
+  }
```

Comment:
You could shift all this along with the task state and useEffect in a custom hook. the same with the favorite functions and state/useEffect. This would make you Add.js much cleaner